pkg/deploy/api/types.go

```diff
@@ -292,6 +292,10 @@ type DeploymentConfigSpec struct {
 	// Replicas is the number of desired replicas.
 	Replicas int
 
+	// RevisionHistoryLimit is the number of old ReplicationControllers to retain to allow for rollbacks.
```

Comment:
@smarterclayton I would appreciate your input on this doc. We want to communicate that `RevisionHistoryLimit` is the number of DCs that will be kept from the set of DCs that:
 - have no active pods
 - have had their generation observed
 - are older than the currently active deployment (which is ... ??)